package.json

```diff
@@ -317,6 +317,7 @@
     "@ledgerhq/hw-transport-webhid": "^6.4.1",
     "@types/jszip": "^3.1.6",
     "@types/parse-torrent": "^5.8.3",
+    "@types/react-router-dom": "^5.1.9",
```

Comment:
This PR seems to be missing the `package-lock.json` changes associated with these new dependencies?

Reply:
Added!